### DISCUSSION TITLE
Workflow name suddenly "(Unknown event)"

### DISCUSSION THREAD

**Discussion by chingc** [community/community#26457] [Actions]
<p>I have a few workflows that are all suddenly showing up as “(Unknown event)”.  This isn’t just for new runs but for every run in its history, even the ones I know had a proper name.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/ffa905971bba76605ddb0d1f9b58c778c505e158.png" data-download-href="/uploads/short-url/AtFSIDyrMk8Ll4qg7b6yZmgVLok.png?dl=1" title="Image%202020-04-27%20at%2017.25.56" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/471856/181097895-d3544e35-0d2c-4000-a9d1-b14ea0304a02.png" alt="Image%202020-04-27%20at%2017.25.56" data-base62-sha1="AtFSIDyrMk8Ll4qg7b6yZmgVLok" width="387" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/ffa905971bba76605ddb0d1f9b58c778c505e158_2_387x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/f/ffa905971bba76605ddb0d1f9b58c778c505e158_2_580x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/f/ffa905971bba76605ddb0d1f9b58c778c505e158.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/ffa905971bba76605ddb0d1f9b58c778c505e158_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Image%202020-04-27%20at%2017.25.56</span><span class="informations">730×942 48.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by andymckay** [comment, marked as answer]
<p>Thanks for the heads up, we are working on it and it should be fixed up shortly.</p>
